### DISCUSSION TITLE
rector phpstan requirement

### DISCUSSION THREAD

**Discussion by staabm** [rectorphp/rector#6597] [Q&A] [answered]
on https://packagist.org/packages/rector/rector#0.11.31 we can see rector requires a certain phpstan version.

is this really a requirement, or does rector itself (since scoped) already contain the phpstan version it requires?

**Comment by TomasVotruba** [comment, marked as answer]
It's a required package, as you can see in `rector/rector`: https://github.com/rectorphp/rector/tree/main/vendor

For PHP 7.0- we would have to scope it, as PHPStan itself is in PHP 7.1 syntax. But that's not the case now
